providers/hns/hns_roce_u_verbs.c

```diff
@@ -270,14 +270,9 @@ static int align_qp_size(int req)
 	return nent;
 }
 
-static int align_srq_size(int req)
+static int align_queue_size(int req)
```

Comment:
unsigned values should have unsigned types

Reply:
OK, thanks for your comments :)
I have re-pushed a new version.